### PR TITLE
text added mentioning detailed explanation urls

### PR DESCRIPTION
**Pull request by Malind19**

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/usmt/usmt-migrate-user-accounts.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ By default, all users are migrated. The only way to specify which users to inclu
          
 
 ## <a href="" id="bkmk-migratetwo"></a>To migrate two domain accounts (User1 and User2)
-
+Links to detailed explanations on commands are available in Related Topics section.
```
Comment (JohanFreelancer9) [inline windows/deployment/usmt/usmt-migrate-user-accounts.md]: Links to detailed explanations **of** commands are available in **the** Related Topics section.

**Comment by mypil** [comment]
@Malind19  - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions he made. Thank you.

**Comment by Malind19** [comment]
@mypil  Changes are added as a different commit to this same PR.

**Comment by mypil** [comment]
@greg-lindsay - Can you please assist to check, approve and merge the content for this PR based on issue #768?

Thank you.

**Comment by mypil** [comment]
@greg-lindsay - I would like to follow up. can you please assist to check, approve and merge the content for this PR based on issue #768?

**Comment by mypil** [comment]
@AndreaBarr - would you be able to merge the content of this PR based on issue #768? Thank you.

**Comment by AndreaBarr** [comment]
@Malind19 The suggested edits were not made to this PR. Please make those suggested edits so we can review and merge this PR.

**Comment by Malind19** [comment]
@AndreaBarr , The grammar fixes are done in the second commit related to this PR. Kindly check.

**Pull request merged**